react/features/unsupported-browser/components/NoMobileApp.js

```diff
@@ -0,0 +1,34 @@
+/* @flow */
+
+import React, { Component } from 'react';
+
+declare var interfaceConfig: Object;
+
+/**
+ * React component representing no mobile page.
```

Comment:
Just a type correction, this should say: React component representing no mobile app page.